third_party/3/jack/__init__.pyi

```diff
@@ -0,0 +1,266 @@
+from typing import Any, Callable, Dict, Generator, Iterable, Iterator, List, Optional, Sequence, Tuple, Union, overload
+
+NDArray = Any  # FIXME: no typings for numpy arrays
+
+JackPositionT = Any
+
+class CBufferType:
+    @overload
+    def __getitem__(self, key: int) -> str: ...
```

Comment:
The str/bytes difference here looks odd. Maybe we need cffi stubs with a more precise type though.

Reply:
I got that from the [cffi docs](https://cffi.readthedocs.io/en/latest/ref.html#ffi-buffer-ffi-from-buffer):

> `buf[:]` or `bytes(buf)`: copy data out of the buffer, returning a regular byte string (or buf[start:end] for a part)
> `buf[:] = newstr`: copy data into the buffer (or buf[start:end] = newstr)
> `len(buf)`, `buf[index]`, `buf[index] = newchar`: access as a sequence of characters.

It's entirely possible I've misinterpreted them though